contrib/ros/robots/sawyer.py

```diff
@@ -104,15 +104,17 @@ def action_space(self):
         # For our use
         # joints[3:10]
         # set the action space depending on different control modes
+        joint_limits = rospy.wait_for_message('/robot/joint_limits',
```

Comment:
Why would you retrieve the joint limits on every call to this function (blocking on a ros message), rather than getting them once and caching them?